etor/src/main/resources/transformation_definitions.json

```diff
@@ -74,38 +74,38 @@
       ]
     },
     {
-      "name": "ucsdOruRemoveMessageTypeStructure",
-      "description": "Removes UCSD ORU's Message Structure (MSH-9.3) from the Message Type (MSH-9)",
+      "name": "ucsdOruUpdateReceivingFacilityWithOrderingFacilityIdentifier",
+      "description": "Updates UCSD ORU Receiving Facility (MSH-6) to value in ORC-21.10 and remove content from MSH-6.2 and MSH-6.3",
       "message": "",
       "conditions": [
         "Bundle.entry.resource.ofType(MessageHeader).event.code = 'R01'",
         "Bundle.entry.resource.ofType(MessageHeader).destination.receiver.resolve().identifier.where(extension.value = 'HD.2,HD.3').value = 'UCSD'"
       ],
       "rules": [
         {
-          "name": "RemoveMessageTypeStructure",
+          "name": "UpdateReceivingFacilityWithOrderingFacilityIdentifier",
           "args": { }
         }
       ]
     },
     {
-      "name": "ucsdOruSwapPlacerOrderAndGroupNumbers",
-      "description": "Swaps UCSD ORU's Placer Order Number (ORC-2) and Placer Group Number (ORC-4)",
+      "name": "ucsdOruRemoveMessageTypeStructure",
+      "description": "Removes UCSD ORU Message Structure (MSH-9.3) from the Message Type (MSH-9)",
       "message": "",
       "conditions": [
         "Bundle.entry.resource.ofType(MessageHeader).event.code = 'R01'",
         "Bundle.entry.resource.ofType(MessageHeader).destination.receiver.resolve().identifier.where(extension.value = 'HD.2,HD.3').value = 'UCSD'"
       ],
       "rules": [
         {
-          "name": "SwapPlacerOrderAndGroupNumbers",
```

Comment:
I see some rules in this file got renamed.  Does this mean that some Java transformations were renamed but they were accidentally not renamed here until now?

Reply:
They were not renamed in this PR, I just re ordered them to group them by resource

Reply:
Oh, ummm, right.  Then what is happening to `SwapPlacerOrderAndGroupNumbers`?  It isn't in `transformation_definitions.json` at all now.  Is it unused?

Reply:
Good catch! It seems it good removed by mistake? I'll check